.eslintrc.js

```diff
@@ -11,5 +11,8 @@ module.exports = {
     "promise",
     "standard",
     "react"
-  ]
+  ],
+  "rules": {
+    "yoda": [2, "never", { "exceptRange": true }]
+  }
```

Comment:
I really like using `if (1 <= variable && variable <= 5)` as a way to check if a variable is in a given range. This is a small change to standard's eslint rules to allow this.